### ISSUE TITLE
swww AUR isn't compiling correctly. Installation fails

### ISSUE THREAD

**Issue by mylinuxforwork** [LGFae/swww#241]
Hi.

Users have reported that the installation of the current AUR of swww 0.8.2-2 fails due to compiling issues.

```
error: non-binding let on a synchronization lock
   --> daemon/src/wallpaper.rs:146:21
    |
146 |         let (inner, _) = self.lock();
    |                     ^ this lock is not assigned to a binding and is immediately dropped
    |
    = help: consider immediately dropping the value using `drop(..)` after the `let` statement
    = note: `#[deny(let_underscore_lock)]` on by default
help: consider binding to an unused variable to avoid immediately dropping the value
    |
146 |         let (inner, _unused) = self.lock();
    |                     ~~~~~~~

error: could not compile `swww-daemon` (bin "swww-daemon") due to 1 previous error
warning: build failed, waiting for other jobs to finish...
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: swww-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
swww - exit status 4
```

I have suggested to update to swww-git with:

```
sudo rm /usr/lib/debug/usr/bin/swww-daemon.debug (if the file exits)
sudo rm /usr/lib/debug/usr/bin/swww.debug (if the file exits)
yay -S swww-git

```

When is the fixed AUR available?

Thanks.

**Comment by aer-i** [comment]
Same problem

**Comment by LGFae** [comment]
This is a duplicate of #234. It has already been fixed in version `0.9.0`. You can either wait for the AUR to be updated, or use the git version as @RevMirin did.
